source/_integrations/nfandroidtv.markdown

```diff
@@ -1,79 +1,59 @@
 ---
-title: Notifications for Android TV / FireTV
-description: Notifications for Android TV / FireTV
+title: Notifications for Android TV / Fire TV
+description: Notifications for Android TV / Fire TV
 ha_category:
   - Notifications
 ha_release: 0.32
+ha_config_flow: true
 ha_domain: nfandroidtv
 ha_iot_class: Local Push
 ha_platforms:
   - notify
+ha_codeowners:
+  - '@tkdrob'
 ---
 
-Notification platform for [Notifications for Android TV](https://play.google.com/store/apps/details?id=de.cyberdream.androidtv.notifications.google) and [Notifications for FireTV](https://play.google.com/store/apps/details?id=de.cyberdream.firenotifications.google). You can use this plarform to send notifications to your Android TV device. An overlay with the message content will appear for a configurable amount of seconds and then disapper again. Sending images (e.g., security cam) is supported too.
+Notification platform for [Notifications for Android TV](https://play.google.com/store/apps/details?id=de.cyberdream.androidtv.notifications.google) and [Notifications for Fire TV](https://www.amazon.com/Christian-Fees-Notifications-for-Fire/dp/B00OESCXEK). You can use this platform to send notifications to your Android TV device. An overlay with the message content will appear for a configurable amount of seconds and then disappear again. Sending images (e.g., security cam) is supported too.
 
 The notifications are in the global scope of your Android TV device. They will be displayed regardless of which application is running.
 
 When setting this up be aware, that there are two apps: one for your smartphone to send notifications (not required for this platform) and one for your Android TV device to receive the notifications. The app available in the store of your Android TV device is the one that is needed to display notifications sent from Home Assistant. The In-App purchases only apply to the client for Android smartphones, so there isn't any limit when pushing notifications from Home Assistant.
 
-To enable the notification platform, add the following to your `configuration.yaml` file:
+{% include integrations/config_flow.md %}
 
-```yaml
-# Example configuration.yaml entry
-notify:
-  - platform: nfandroidtv
-    name: Kitchen
-    host: 192.168.1.12
-```
+The following options can be specified inside the data field for the notify service call:
```

Comment:
Please document it as a service as we do on all other integration pages as well.